src/configuration/configuration-base.ts

```diff
@@ -116,44 +127,84 @@ export default class Configuration {
         return cloneDeep(this);
     }
 
-    public get filePath (): string | null {
+    public get filePath (): string | undefined {
         return this._filePath;
     }
 
+    public get defaultPaths (): string[] | undefined {
+        return this._defaultPaths;
+    }
+
     public async _load (): Promise<null | object> {
-        if (!this.filePath)
+        if (!this.defaultPaths || !this.defaultPaths.length)
             return null;
 
-        if (!await this._isConfigurationFileExists())
-            return null;
+        const options = await Promise.all(this.defaultPaths.map(async filePath => {
+            if (!await this._isConfigurationFileExists(filePath))
+                return null;
 
-        const configurationFileContent = await this._readConfigurationFileContent();
+            if (this._isJSConfiguration(filePath))
+                return this._readJsConfigurationFileContent(filePath);
 
-        if (!configurationFileContent)
-            return null;
+            const configurationFileContent = await this._readConfigurationFileContent(filePath);
+
+            if (!configurationFileContent)
+                return null;
+
+            return this._parseConfigurationFileContent(configurationFileContent);
+        }));
 
-        return this._parseConfigurationFileContent(configurationFileContent);
+        return options.reduce((result, option, index) => {
+            if (option && result)
+                Configuration._showConsoleWarning(WARNING_MESSAGES.multipleConfigurationFilesFound);
```

Comment:
Build the configuration priority string from [here](https://github.com/DevExpress/testcafe/pull/6319/files#diff-c13e204be9af6520044d50add54e47dc2ebdd5dab1ab19de5650d24361997879R42) and pass it to the warning